test/OpenTelemetry.Instrumentation.AspNetCore.Tests/IncomingRequestsCollectionsIsAccordingToTheSpecTests.cs

```diff
@@ -106,7 +107,7 @@ public IncomingRequestsCollectionsIsAccordingToTheSpecTests(WebApplicationFactor
 
             if (statusCode == 503)
             {
-                Assert.Equal(Status.Error, activity.GetStatus());
+                Assert.Equal(Status.Error.StatusCode, activity.GetStatus().StatusCode);
```

Comment:
please add a test for this.options.RecordException as well. and i think this is good.

Reply:
done!